### PR TITLE
perf: batch tmux show

### PR DESCRIPTION
**Pull request by vdbe**
Try 2

### CODE REVIEW

**Comment by Bjorn-Eric-Abr** [comment]
Hello! Thanks for a great plugin. I forked the `main` branch of the repo to make some customisations (as you do) and found something odd.
After making my changes I wanted to customise Gitmux and it only showed the Git icon and nothing else. Reforking the whole repo (not just `main` as per Github defaults) and using `#latest` on my forked branch and the customisations work, as they do with the `#latest` (tag 0.1.0) in this repo.

**TLDR;** I think this commit makes the Gitmux customisations not work. Just wanted to give a heads up, if this is known plz ignore :)

**Comment by vdbe** [comment]
@Bjorn-Eric-Abr Can you create an issue with the config used, default options from `gitmux` seem to work on the main branch (don't use it myself so don't know if anything is missing)?
image

**Comment by Bjorn-Eric-Abr** [comment]
Thanks for the fast reply!
Same for me, default works but customisation does not. I'll create an issue.
